subsys/bluetooth/controller/Kconfig

```diff
@@ -750,4 +750,18 @@ config BT_MAYFLY_YIELD_AFTER_CALL
 	  If set to 'n', all pending mayflies for callee are executed before
 	  yielding
 
+config BT_CTLR_USER_EXT
+	prompt "Enable proprietary extensions in Controller"
+	bool
+	help
+	  Catch-all for enabling proprietary event types in Controller behavior.
+
+config BT_CTLR_USER_EVT_RANGE
+	int "Range of event contants reserved for proprietary event types"
+	default 5
```

Comment:
This should not default to 5:

```
default 0
default 5 if BT_CTLR_USER_EXT
```

Reply:
I have implemented changes you suggested below (i.e. dependency on BT_CTLR_USER_EXT). In that case we need a default that is larger than 0.